app/http.php

```diff
@@ -711,15 +712,15 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
     ->param('source', '', new Text(0), 'Path to source files.', true)
     ->param('entrypoint', '', new Text(256), 'Entrypoint of the code file.', true)
     ->param('variables', [], new Assoc(), 'Environment variables passed into runtime.', true)
-    ->param('cpus', 1, new Integer(), 'Container CPU.', true)
+    ->param('cpus', 1.0, new FloatValidator(), 'Container CPU.', true)
     ->param('memory', 512, new Integer(), 'Container RAM memory.', true)
     ->param('version', 'v3', new WhiteList(['v2', 'v3']), 'Runtime Open Runtime version.', true)
     ->param('runtimeEntrypoint', '', new Text(1024, 0), 'Commands to run when creating a container. Maximum of 100 commands are allowed, each 1024 characters long.', true)
     ->inject('activeRuntimes')
     ->inject('response')
     ->inject('log')
     ->action(
-        function (string $runtimeId, ?string $payload, string $path, string $method, array $headers, int $timeout, string $image, string $source, string $entrypoint, array $variables, int $cpus, int $memory, string $version, string $runtimeEntrypoint, Table $activeRuntimes, Response $response, Log $log) {
+        function (string $runtimeId, ?string $payload, string $path, string $method, array $headers, int $timeout, string $image, string $source, string $entrypoint, array $variables, float $cpus, int $memory, string $version, string $runtimeEntrypoint, Table $activeRuntimes, Response $response, Log $log) {
```

Comment:
We use $cpus in `setCpu()`. that is https://github.com/utopia-php/orchestration/blob/main/src/Orchestration/Orchestration.php#L217

Notice it accepts int. We need to change orchestration library to support float too

Reply:
Yeap we updated it 👍  Release 0.10.1 of it